src/app/home/home.component.ts

```diff
@@ -1,10 +1,24 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {Router} from "@angular/router";
 
 @Component({
   selector: 'bmm-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
-export class HomeComponent {
+export class HomeComponent implements OnInit{
+  constructor(
+    private readonly http: HttpClient,
+    private readonly _router: Router
+  ) {
+  }
+  ngOnInit(): void {
+    const backendUrl = 'http://localhost:8080'; // Change this to your backend URL
+    const apiUrl = `${backendUrl}/currentseason`;
```

Comment:
The "apiUrl" should be the backend URL, and it can be imported from the environment, see example in season.service.ts.
With the environment, you can configure your local backend (default is localhost:8080 which seems to be matching), while for production, it can be set to whatever is suitable for production prior to building the project.